src/rules/no-snapshots.js

```diff
@@ -0,0 +1,37 @@
+import { getDocsUrl } from './util';
+
+export default {
+  meta: {
+    docs: {
+      description: 'Disallow snapshots',
+      uri: getDocsUrl(__filename),
+    },
+    messages: {
+      noSnapshots:
+        'Do not use {{method}} or related methods that generate snapshots.',
```

Comment:
Could this have inner spaces, to be consistent with the other rules? (i.e `{{ method }}`)